sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -296,6 +296,15 @@ class ColumnExpressionSuite extends QueryTest {
     checkAnswer(testData.filter($"a".between($"b", $"c")), expectAnswer)
   }
 
+  test("in") {
+    var expectedAnswer = testData2.collect().toSeq.filter(r => r.getInt(0) == 1 || r.getInt(0) == 2)
+    checkAnswer(testData2.filter($"a" in(1, 2)), expectedAnswer)
```

Comment:
I think we prefer to have explicitly defined test data in the test cases. The testData/testData1 etc are from old test cases.

Take a look at test case "self join explode" for an example.